modules/installation-launching-installer.adoc

```diff
@@ -341,6 +362,11 @@ program creates. Both are required to delete the cluster.
 ifdef::aws[]
 . Optional: Remove or disable the `AdministratorAccess` policy from the IAM
 account that you used to install the cluster.
++
+[NOTE]
+====
+The elevated permissions provided by the `AdministratorAccess` policy are only required during installation.
```

Comment:
```suggestion
The elevated permissions provided by the `AdministratorAccess` policy are required only during installation.
```

Reply:
Thank you. I don't yet understand the syntactical principal behind this recommendation but I have made the change. I will look into this.